### ISSUE TITLE
Refactor staking test modules into own files

### ISSUE THREAD

**Issue by gpestana** [paritytech/polkadot-sdk#3178]
The staking `tests.rs` has about 7k LOC. Most of the tests are nicely organised into modules, consider refactor each mod into a new file for readability.

**Comment by gpestana** [comment]
ftr, @vicenterocha is interested in picking up this issue 🚀

**Comment by gpestana** [comment]
Note: this refactoring is probably better suited to be done as part of the multi-block revamp and once https://github.com/paritytech/polkadot-sdk/pull/1933 is merged.

**Comment by rossbulat** [comment]
This is now stale as work is underway to tidy up the runtimes post AH migration (https://github.com/paritytech/polkadot-sdk/pull/11940)

@vicenterocha What do you think about closing this issue now? My concern is that it may otherwise give contributors the impression that it should be worked on. Checking `substrate/frame/staking-async/src/tests` also, looks like tests have already been modularised.

cc @Ank4n

**Comment by sigurpol** [comment]
I fully agree with you @rossbulat  as per my comment [here](https://github.com/paritytech/polkadot-sdk/pull/11129#issuecomment-4649328153) and I am in favor of closing this issue.
Tests on staking-async can surely benefit of extra love and can be improved, but at least the modularization issue seems indeed gone.
